src/Blazor.Gitter.Core/Components/Shared/ChatRoomList.razor

```diff
@@ -24,13 +24,4 @@
 				}
 			</ul>
 		}
-		else

```

Comment:
Why has this section been removed?

Reply:
Temporary - For test purposes, will restore before we are ready for merge

Reply:
Restored